test/ecto_subscribe_test.exs

```diff
@@ -0,0 +1,32 @@
+import Ecto.Subscribe.Api
+
+import Ecto.Query
+
+defmodule TestModel do
+  use Ecto.Schema
+  @primary_key false
+  subscribe(repo: EctoIt.Repo)
```

Comment:
If you will do:
require Ecto.Subscribe.Api

and
Ecto.Subscribe.Api.subscribe

it doen't working anymore.